tests/unit/contracts/graph/test_nodes.py

```diff
@@ -236,10 +236,11 @@ def test_basic_compiled_model(basic_compiled_dict, basic_compiled_model):
     assert node.is_ephemeral is False
 
 
-def test_invalid_extra_fields_model(minimal_uncompiled_dict):
-    bad_extra = minimal_uncompiled_dict
-    bad_extra["notvalid"] = "nope"
-    assert_fails_validation(bad_extra, ModelNode)
+def test_extra_fields_model_okay(minimal_uncompiled_dict):
+    extra = minimal_uncompiled_dict
+    extra["notvalid"] = "nope"
+    # Model still load fine with extra fields
+    ModelNode.from_dict(extra)
```

Comment:
might be overkill, but we could also test that the ModelNode does not have the extra attribute on the class, or any `_extra` property.

Reply:
I guess this test may not be necessary with the new `validate` implementation though...